.github/workflows/prod.yml

```diff
@@ -1,12 +1,12 @@
-name: CI
+name: Production
 
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master, develop ]
+    branches: [ master ]
```

Comment:
Switch this off on pull-request and push? This should be triggered only when triggered manually

Reply:
Should I remove push/pull triggers for the staging.yml as well?